Services/IBlobStorage.cs

```diff
@@ -0,0 +1,44 @@
+using System.IO;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+
+namespace NosAyudamos
+{
+    public interface IBlobStorage
+    {
+        Task UploadAsync(byte[] bytes, string containerName, string blobName);
+    }
+
+    public class BlobStorage : IBlobStorage
+    {
+        private readonly IEnviroment enviroment;
+
+        public BlobStorage(IEnviroment enviroment)
+        {
+            this.enviroment = enviroment;
+        }
+
+        public async Task UploadAsync(byte[] bytes, string containerName, string blobName)
+        {
+            var blobServiceClient = CreateBlobServiceClient();
+            var containerClient = blobServiceClient.GetBlobContainerClient(containerName);
+
+            if(!await containerClient.ExistsAsync())
```

Comment:
whitespaces are wrong and that fails the build now. plz install `dotnet tool install -g dotnet-format` and then run it in-place with `dotnet format`. VSCode should have picked the formats from `.editorconfig` but apparently it missed some :/